### PR TITLE
87 integrate scss

### PR DESCRIPTION
**Pull request by Cozloschi**

### CODE REVIEW

File: src/grunt/presswise.scss [hunk 1]
```diff
@@ -0,0 +1,728 @@
...
+  text-indent: -9999px;
+  width: 48px;
+  height: 48px;
+}
+.timer-loader:not(:required)::before {
+  -moz-animation: timer-loader 1250ms infinite linear;
+  -webkit-animation: timer-loader 1250ms infinite linear;
+  animation: timer-loader 1250ms infinite linear;
+  -moz-transform-origin: 3px 3px;
+  -ms-transform-origin: 3px 3px;
```
Comment (vtemian) [inline src/grunt/presswise.scss]: use the same identation and variables where you can

**Comment by vtemian** [comment]
split scss in multiple files based on common functionalities

File: provisioning/deploy.yml [hunk 2]
```diff
@@ -2,17 +2,33 @@
...
-     git: repo={{ project_repo }} dest={{ project_home }}/{{ project_dir }} accept_hostkey=true force=yes
-      
+     git: repo={{ project_repo }} dest={{ project_home }}/{{ project_dir }} accept_hostkey=true force=yes version=87-integrate-scss
+
    - name: Install Requirements.txt
      pip:  requirements={{project_home}}/{{ project_dir }}/requirements.txt virtualenv={{project_home}}
      sudo: yes
- 
+
    #- name: Migrate database
```
Comment (vtemian) [inline provisioning/deploy.yml]: Uncomment this one

File: src/grunt/_general.scss [hunk 3]
```diff
@@ -0,0 +1,782 @@
...
+$icon-thumbs-up: "\f164";
+$icon-ticket: "\f145";
+$icon-times: "\f00d";
+$icon-times-circle: "\f057";
+$icon-times-circle-o: "\f05c";
+$icon-tint: "\f043";
+$icon-toggle-down: "\f150";
+$icon-toggle-left: "\f191";
+$icon-toggle-off: "\f204";
+$icon-toggle-on: "\f205";
```
Comment (vtemian) [inline src/grunt/_general.scss]: Move this in `icons.scss`

File: src/grunt/_general.scss [hunk 4]
```diff
@@ -0,0 +1,782 @@
...
+$fa-font-path: "../fonts" !default;
+$fa-version: "4.4.0" !default;
+
+$font__main: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+$font__heading: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+$font__line-height-body: 1.5;
+$font__line-height-pre: 1.6;
+
+
+@function strip-unit($num) {
```
Comment (vtemian) [inline src/grunt/_general.scss]: Move all the functions in `functions.scss`

File: src/grunt/_general.scss [hunk 5]
```diff
@@ -0,0 +1,782 @@
...
+    &:before {
+        top: -($height + $gutter);
+    }
+
+    &:after {
+        top: $height + $gutter;
+    }
+}
+
+@mixin burger-parts {
```
Comment (vtemian) [inline src/grunt/_general.scss]: Move all the mixins in `mixins.scss`

**Comment by vtemian** [comment]
Rename `_general.scss` in `general.scss`

File: src/grunt/content.scss [hunk 6]
```diff
@@ -0,0 +1,336 @@
+.body {
```
Comment (vtemian) [inline src/grunt/content.scss]: This one is pointless

File: src/grunt/presswise.scss [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+@mixin default-overlay-input {
+  @include default-input();
+  background: transparent;
+  border: 1px solid transparent;
+  left: 0;
+  right: 0;
+  width: auto;
+}
+
+@mixin default-button($background: $color__main, $border: $color__main) {
```
Comment (vtemian) [inline src/grunt/presswise.scss]: Move mixins in `mixins.scss` and let only variables to be defined here.

Comment (vtemian) [inline src/grunt/presswise.scss]: Also, you can move all variables from `general.scss` here.

**Comment by vtemian** [comment]
Remove the css file from git tree.

**Comment by Cozloschi** [comment]
@vtemian

**Comment by vtemian** [comment]
Conflicts with master

**Pull request merged**